Users/Tutorial.rst

```diff
@@ -314,6 +314,20 @@ If you put an ``all`` instead of the bear names directly after the
 ``ignore``/``ignoring`` keyword, the results of all bears affecting
 those lines will be ignored.
 
+ If you've used another linter in the past, you don't have to change your pre-existing code with the 'noqa' keywords to 'ignore' as the following examples work aswell:
+
+::
+    # Noqa LineLengthBear
```

Comment:
The number of spaces is misaligning this code block.
Use 4 spaces for both lines?